internal/alpini/common/alpini-common-base/build.gradle

```diff
@@ -4,7 +4,7 @@ configurations {
 
 dependencies {
   testCompile libraries.testng
-  compile project(':internal:alpini:common:common-const')
+  compile project(':internal:alpini:common:alpini-common-const')
```

Comment:
`compile` dependency is deprecated, should use `implementation` instead.

Reply:
Done. This led to a bunch of transitive failures - but I think the dependencies are in a better shape now